airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -98,14 +99,17 @@ def facebook_ads_config(self) -> Dict:
         if missing_keys:
             message = f"{missing_keys} fields are missing"
             raise AirflowException(message)
+        # Solve a way to check in bulk_facebook_report to handle list and dict return there
+        if type(config["account_id"]) is not list:
```

Comment:
```suggestion
        if isinstance(config["account_id"], list):
```

This `is_backward` logic is nebulous to me though. Is this “backward if `account_id` is specified” behaviour specified in Facebook’s API documentation?

Reply:
Normally, the `account_id` was given as a string in the Facebook Ads Connection in the Airflow, with this check it is controlling rather the connection is the old version or is a new version which is an array of `account_id`. Also, I have updated this part regarding your suggestion.